### ISSUE TITLE
Investigate `get_host_by_name` panick in debug mode if zero or more than 1 aliases

### ISSUE THREAD

**Issue by petski** [petski/nss-docker-ng#5]
```
thread '<unnamed>' panicked at /home/.../.cargo/registry/src/index.crates.io-6f17d22bba15001f/libnss-0.5.0/src/host.rs:60:21:
misaligned pointer dereference: address must be a multiple of 0x8 but is 0x56257c28ca94
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
thread caused non-unwinding panic. aborting.
Aborted (core dumped)
```

See https://stackoverflow.com/questions/76647616/facing-unexpected-misaligned-pointer-dereference-address-must-be-a-multiple-of

As a work-around, you can add this to your `Cargo.toml`:

```toml
[profile.dev]
debug-assertions = false
```

**Comment by petski** [comment]
Fixed in https://github.com/csnewman/libnss-rs/commit/6046c8d865120563baf4d644d72ebb5cd01249a0

Asked for a version bump in https://github.com/csnewman/libnss-rs/pull/16#issuecomment-1947517665

For people needing this, use for now:

```toml
libnss = { git ="https://github.com/csnewman/libnss-rs", rev = "870253e" }
```

**Comment by petski** [comment]
Fixed with #7
